app/assets/templates/header.html.erb

```diff
@@ -7,11 +7,7 @@
   </a>
 
   <div class="collapse navbar-collapse" id="navbarNav">
-    <ul ng-if="$ctrl.is_admin"  class="navbar-nav">
-      <li class="nav-item active">
-        <%= link_to 'Dashboard', admin_root_path, {class: 'nav-link'} %>
-      </li>
-    </ul>
+
```

Comment:
are we getting rid of the dashboard button completely? so now if an admin wants to navigate from map or users/documents index page back to admin home, is url the only way to do that now?